pendulum_msgs/msg/RttestResults.msg

```diff
@@ -0,0 +1,11 @@
+builtin_interfaces/Time stamp
+
+pendulum_msgs/JointCommand command
+pendulum_msgs/JointState state
```

Comment:
You don't have to mention the same package name explicitly. I am not sure which option we should prefer:
* either skip it when possible for simplifying the file and avoiding repetition of the package name
* always use fully qualified names including the package name to be explicit